v3/src/components/tool-shelf/tool-shelf.scss

```diff
@@ -0,0 +1,26 @@
+@import "../vars.scss/";
+
+.tool-shelf {
+  height: $tool-shelf-height;
+  width: 100%;
+  background-color: #ececec;
+  border-bottom: 1px solid #dcdbd8;
+  font-family: 'MuseoSans-500', 'sans-serif';
```

Comment:
TODO: guarantee that MuseoSans is available, either by including it in the project bundle or linking to it via CDN.